pkg/controller/controller.go

```diff
@@ -61,28 +62,37 @@ func New(cfg Config) *Controller {
 	}
 }
 
-func (c *Controller) handleClusterEvent(event *Event) error {
+// handleClusterEvent returns ignored true if cluster isn't managed by this instance.
```

Comment:
returns ~~ignored~~ true if cluster is ignored (not managed) by this instance.

Reply:
@guilhem fix this?